contrib/syslog-ng-debun

```diff
@@ -1679,6 +1712,8 @@ if [ -n "$privacy_mode" ]; then
 	acquire_syslog_nprv
 else
 	acquire_syslog_all
+	acquire_kafka_properties_file
```

Comment:
I would probably create a wrapper subroutine here for gathering similar property files, not only kafka.
Of course with your current contribution it would be the only subroutine called there, but it could be extended easily later on.

Reply:
Okay. Thanks for review